grumpy-projection/src/main/java/com/github/davidmoten/grumpy/projection/Projector.java

```diff
@@ -1,215 +1,213 @@
-		p.y = (int) Math.round(point2D.getY());
-		return p;
-	}
-
-	public Point2D.Double toPointInSrs(double lat, double lon) {
-		com.vividsolutions.jts.geom.Point point = getGeometryPointInSrs(lat, lon);
-		return new Point2D.Double(point.getX(), point.getY());
-	}
-
-	public Point2D.Double toPoint2D(double lat, double lon) {
-		Coordinate coordinate = new Coordinate(lon, lat);
-		com.vividsolutions.jts.geom.Point point = geometryFactory.createPoint(coordinate);
-		try {
-			point = (com.vividsolutions.jts.geom.Point) JTS.transform(point, transform);
-
-			double proportionX;
-			if (point.getX() > bounds.getMaxX() || point.getX() < bounds.getMinX()) {
-				// assume the maxX occurs at longitude 180 (true for EPSG 3857
-				// spherical mercator) but maybe not true for other projections?
-				Coordinate c = new Coordinate(180, 0);
-				com.vividsolutions.jts.geom.Point pt = (com.vividsolutions.jts.geom.Point) JTS
-				        .transform(geometryFactory.createPoint(c), transform);
-				double maximumX = pt.getX();
-				if (point.getX() > bounds.getMaxX())
-					proportionX = (point.getX() - 2 * maximumX - bounds.getMinX())
-					        / (bounds.getMaxX() - bounds.getMinX());
-				else
-					proportionX = (point.getX() + 2 * maximumX - bounds.getMinX())
-					        / (bounds.getMaxX() - bounds.getMinX());
-			} else {
-				proportionX = (point.getX() - bounds.getMinX())
-				        / (bounds.getMaxX() - bounds.getMinX());
-			}
-			double proportionY = (bounds.getMaxY() - point.getY())
-			        / (bounds.getMaxY() - bounds.getMinY());
-			Point2D.Double point2D = new Point2D.Double(proportionX * target.getWidth(),
-			        proportionY * target.getHeight());
-			return point2D;
-		} catch (MismatchedDimensionException e) {
-			throw new RuntimeException(e);
-		} catch (TransformException e) {
-			throw new RuntimeException(e);
-		}
-	}
-
-	public Position toPosition(double targetX, double targetY) {
-		double proportionX = targetX / target.getWidth();
-		double proportionY = targetY / target.getHeight();
-		double x = proportionX * (bounds.getMaxX() - bounds.getMinX()) + bounds.getMinX();
-		double y = bounds.getMaxY() - proportionY * (bounds.getMaxY() - bounds.getMinY());
-		Coordinate coordinate = new Coordinate(x, y);
-		com.vividsolutions.jts.geom.Point point = geometryFactory.createPoint(coordinate);
-		try {
-			point = (com.vividsolutions.jts.geom.Point) JTS.transform(point, transform.inverse());
-		} catch (MismatchedDimensionException e) {
-			throw new RuntimeException(e);
-		} catch (TransformException e) {
-			throw new RuntimeException(e);
-		}
-		return new Position(point.getY(), point.getX());
-	}
-
-	public Position toPositionFromSrs(double x, double y) {
-		return FeatureUtil.convertToLatLon(x, y, bounds.getSrs());
-	}
-
-	@Override
-	public String toString() {
-		return "ProjectorImpl [target=" + target + ", bounds=" + bounds + "]";
-	}
+    private static final AtomicBoolean initialized = new AtomicBoolean(false);
```

Comment:
This looks like a bunch of white space changes. Please don't do this because it complicates review.

Reply:
Ahh, yes. There's en entire file with white space change. Didn't realise it. I will send another PR with neater changes.

Reply:
Don't make another PR, just push another commit to this one with the corrections.

Reply:
Alright

Reply:
This is addressed.